readme.md

```diff
@@ -13,6 +13,7 @@ Multi-platform and multi-language.
 ## Why Use Renovate?
 
 - Receive automated Pull Requests whenever dependencies need updating
+- Get replacement PRs to migrate from a deprecated dependency to the community suggested replacement (npm packages only)
```

Comment:
Considering it's npm only, and has only a handful of packages so far, I don't think it's worthy selling point #2

Reply:
Want me to move it down the list then?